raiden/transfer/channel.py

```diff
@@ -1334,8 +1339,11 @@ def create_sendexpiredlock(
     nonce = get_next_nonce(sender_end_state)
     locked_amount = get_amount_locked(sender_end_state)
     balance_proof = sender_end_state.balance_proof
-    updated_locked_amount = locked_amount - locked_lock.amount
-    transferred_amount = balance_proof.transferred_amount
+    updated_locked_amount: typing.TokenAmount = locked_amount - locked_lock.amount
+    if balance_proof:
```

Comment:
Hmm here I believe the `sender_end_state` should always contain a balance_proof since it's `our_state` and this is only generated for a lock expiration so there should be a balance proof. So I would replace this with an assertion.

```python
assert balance_proof, 'The sender end state should always have a balance proof at this point'
```

Reply:
I see. There has to be a lock before it expires.

Reply:
Done https://github.com/raiden-network/raiden/pull/3214/files#diff-fe7772da6084a3493b31e88fbf7f6d55R1345

Reply:
@pirapira Since it's an assertion can you change `is` to `should be`? This is the message you will see if the assertion fails.

Reply:
You're absolutely right.